quickstart/index.md

```diff
@@ -74,13 +74,13 @@ The API is easy to use. From a CoreOS machine, you can simply use curl to set an
 Set a key `message` with value `Hello world`:
 
 ```sh
-curl -L http://127.0.0.1:4001/v2/keys/message -XPUT -d value="Hello world"
+curl -L http://127.0.0.1:2379/v2/keys/message -XPUT -d value="Hello world"
```

Comment:
Looks like we completely forgot to update the quick start to use etcd2. Can you also change the cloud-config exmaple above to use etcd2? It's currently:

```
coreos:
  units:
    - name: etcd.service
```